python/lsst/ts/rubintv/main.py

```diff
@@ -97,6 +98,7 @@ async def add_cache_control_header(
         response.headers["Cache-Control"] = "public, max-age=3600, must-revalidate"
         return response
 
+<<<<<<< HEAD
```

Comment:
Looks like you have an incomplete rebase/merge conflict to resolve.

Reply:
Well, that's embarassing! I don't how this even got past pre-commit... 🙄